napari_clusters_plotter/_plotter.py

```diff
@@ -365,12 +366,14 @@ def run(self, properties, plot_x_axis_name, plot_y_axis_name, plot_cluster_name=
                 properties.keys()):
             self.cluster_ids = properties[plot_cluster_name]
 
-            color = ['#ff7f0e', '#1f77b4', '#2ca02c', '#d62728', '#9467bd', '#8c564b', '#e377c2', '#7f7f7f', '#bcbd22',
-                     '#17becf']
+            colors = pd.read_csv('Nice Colourmap Set3 (256) hex.csv')['colours'].to_numpy()
```

Comment:
This is still old code which unfortunately does not work.... I have already fixed in branch: implementing_dictionary_cluster_labelling(Issue-#21).

Reply:
Yeah I know, but I had to include this change in this branch because in the commit where you make the change to fix hdbscan you also committed this change, and I couldn't find a way to split a commit into two :( 
It will all be merged in the end, no worries!